net-nntp/suck/suck-4.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools
+
+DESCRIPTION="Grab news from a remote NNTP server and feed them to another"
+HOMEPAGE="https://lazarus-pkgs.github.io/lazarus-pkgs/suck.html"
+SRC_URI="https://github.com/lazarus-pkgs/suck/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="ssl perl"
+
+DEPEND="sys-libs/db
+	perl? ( dev-lang/perl )
```

Comment:
The perl flag does not seem to do much - part of the problem is that `configure.ac` is searching for `libperl.a`, which does not exist in Gentoo, but even if I change it to `libperl.so` there is no difference in the install image. It does not set a build switch either. Do you have an idea?

Reply:
I'm not sure how it was in suck 4.3.2, probably it had some effect there. In 4.3.3 maybe it can be omitted, I'm still learning about the package. I would let it like this for now and try to investigate later on more, would that be ok?